exploring-ship-logbooks/word_count.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import math
+
+def word_count(data, columns, key_words):
```

Comment:
How are we going to go about picking the key_words? I was thinking this script would do a word count for all the words present in the logbook. Then we could find the important words based on how many times they appeared. Maybe that is another script we should write for next week. We can use that output for the input to this function.